value.go

```diff
@@ -57,8 +57,8 @@ var (
 	reflectTypeError  = reflect.TypeOf((*error)(nil)).Elem()
 )
 
-var intCache [256]Value
-var int64Cache [256]Value
+var intCache [16384]Value
+var int64Cache [16384]Value
```

Comment:
I considered extracting this out into its own variable but decided to keep it consistent/simple

Reply:
16384 was a fairly arbitrary choice, I wanted to keep the memory footprint ~1mb for each cache

Reply:
[opt] I do like the idea of storing the value into a variable.